### ISSUE TITLE
'now alias' command fails if package.json missing but now.json exists with "alias": ... entry

### ISSUE THREAD

**Issue by christian-schulze** [vercel/vercel#330]
I have a javascript/webpack app which on build, compiles a static website into the `dist` folder. The `dist` folder includes `now.json` file with an `alias` entry.

To deploy I cd into the `dist` folder and run `now`, which deploys successfully, but running `now alias` fails with the following error:
```
> Error! Failed to read JSON in ".../dist/package.json"
```

**Comment by timneutkens** [comment]
Looking into it 👍

**Comment by timneutkens** [comment]
Was fixed in #321. Not released yet.

**Comment by christian-schulze** [comment]
FYI this works when specifying a deployment `type` of `static` in `now.json`, e.g.:
```json
{
  "alias": "custom-domain-name",
  "type": "static"
}
```
